test/es6/lambda1.js

```diff
@@ -270,6 +270,52 @@ var tests = [
         }
     },
     {
+        name: "eval('arguments') works correctly within lambdas",
+        body: function () {
+            var arguments = 'not arguments object';
+            assert.areEqual(arguments, (() => eval("arguments"))(), "arguments in lambda should bind to outside arguments since it is not given its own arguments binding");

```

Comment:
To clarify this, should each instance of `arguments` in this line bind to the var called `arguments` or to an actual arguments object?